### DISCUSSION TITLE
Welcome to api-builder-extras Discussions!

### DISCUSSION THREAD

**Discussion by cwiechmann** [Axway-API-Builder-Ext/api-builder-extras#35] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Wibrandus** [comment]
Hi Chris and community. I work as an IT consultant on mostly DevOps contracts. At the moment I am interested in the apigateway-openlogging-elk project due to a starting PoC at my customers site.

**Comment by Wibrandus** [comment]
@cwiechmann can we expect the next release soon? I am in need of the local API lookup file solution for my implementation. My customer site does not use the API manager, only API Gateway nodes and node manager.

**Reply by cwiechmann** [reply]
Hi @Wibrandus, yes - I'm actually working on the next version and targeting for the end of the year. Perhaps, it will be Mid/End of January.

**Comment by cwiechmann** [comment]
I lock this conversation - As questions should appear in new threads.
